src/lib/intlFormat.test.ts

```diff
@@ -253,7 +253,7 @@ describe('intlFormat', () => {
         locale,
       );
 
-      expect(date.trim()).toBe('Jan 16, 2024, 6:34\u202fPM UTC');
+      expect(date.trim()).toBe('Jan 16, 2024, 6:34 PM UTC');
```

Comment:
Did this break with the upgrade of Node.js?

Reply:
Yes, it was changed in node 18.15.0